openlibrary/core/wikidata.py

```diff
@@ -32,14 +42,10 @@ class WikidataEntity:
     labels: dict[str, str]
     descriptions: dict[str, str]
     aliases: dict[str, list[str]]
-    statements: dict[str, dict]
+    statements: dict[str, list[dict]]
     sitelinks: dict[str, dict]
     _updated: datetime  # This is when we fetched the data, not when the entity was changed in Wikidata
```

Comment:
I know this is from a different PR, but would it make sense to call this `_fetched` or `_last_fetched` instead to avoid the ambiguity and need of the comment? (Probably also something for a different PR since, either way, it’s unrelated to adding Wikipedia links.)